atm/VOLKOV/src/main/java/com/mybank/atm/controller/MyBank.java

```diff
@@ -0,0 +1,202 @@
+package com.mybank.atm.controller;
+
+import com.mybank.atm.model.Client;
+import com.mybank.atm.repository.ClientsRepository;
+
+import java.util.List;
+import java.util.Scanner;
+
+public class MyBank extends ATM {
+
+    private Client client;
+    private Client currentClient;
+    private ClientsRepository clientsRepository = new ClientsRepository();
+    private int truePinNum;
+    private int clientId;
+    private int selected;
+    private List<Client> clients = clientsRepository.getClients();
+
+
+    public void startUpATM() {
+        clientsRepository.creatClients();
+        System.out.println("Hello \nWeclcom to MyBank ");
+        cardVarification();
+    }
+
+    public void cardVarification() {
```

Comment:
This method can go into the Validator class, so it is separated from the Bank logic